package.json

```diff
@@ -73,7 +75,7 @@
     "npm-run-all": "^4.1.5",
     "prettier": "^3.0.0",
     "rimraf": "^5.0.1",
-    "tailwindcss": "3.3.2",
+    "tailwindcss": "^3.3.2",
```

Comment:
Thank you for your PR and I'm sorry for the delay, now I'm fully back.

It seems nativewind 2 only works with tailwindcss@3.3.2: https://www.nativewind.dev/quick-starts/expo

I think it's better to fix the version to 3.3.2

Reply:
Commited a new package.json